osquery/tables/system/windows/drivers.cpp

```diff
@@ -48,46 +49,54 @@ const std::string kServiceKeyPath =
 
 static inline void win32LogWARNING(const std::string& msg,
                                    const DWORD code = GetLastError(),
-                                   const std::string& deviceName = "") {
-  LOG(WARNING) << msg << " for device " << deviceName
+                                   const std::string& device_name = "") {
+  LOG(WARNING) << msg << " for device " << device_name
                << ", error code: " + std::to_string(code);
 }
 
+// Unify the image path as systemRoot can contain systemroot and/or system32
+static inline std::string kNormalizeImage(std::string& path) {
+  std::transform(path.begin(), path.end(), path.begin(), ::tolower);
+  char sys_root[MAX_PATH] = {0};
+  GetSystemDirectory(sys_root, MAX_PATH);
+  return sys_root +
+         boost::regex_replace(path, boost::regex("^.*?system32"), "");
```

Comment:
If you think performance matters here, we can avoid regex using find.

Reply:
@guliashvili are you suggesting something like `find("system32") != .end()` then run the `regex_replace`?

Reply:
const string system32("system32");
const auto it =  path.find(system32);
if(it != path.end()){
          sys_root.insert(sys_root.end(), it + system32.size(), path.end());
}

return sys_root;

Definitely looks ugly would vote for not using it if you think it does not matter a lot.